src/main/java/nl/pin/paardenstal/services/StallService.java

```diff
@@ -29,6 +36,10 @@ public List<StallDto> getAllStalls(){
         stalls = stallRepository.findAll();
         for(Stall s: stalls){
             StallDto dto = transferToDto(s);
+            if(s.getHorse() != null){
+                HorseDto horseDto = horseService.transfertoDto(s.getHorse());
```

Comment:
goed dat de HorseDto hieraan toegevoegd is. Mis deze nog in de getStall() methode.